containers/podman-desktop
# chore: make contour file available offline

### What does this PR do?
Makes contour.yaml file generated on demand and permanent part of the repo, available under `extensions/kind/src/resources`. Can be updated by npm task `install:contour`.
### Screenshot/screencast of this PR



### What issues does this PR fix or reference?
#2248 


### How to test this PR?
Kind extension build passes: `yarn build:extensions:kind`

Verify downloading/generating of the file:
Removing the contour available locally: `rm extensions/kind/src/resources`
`yarn build:extensions:kind` -> fails
To download/generate contour:
`yarn --cwd extensions/kind run install:contour`
will be available on `extensions/kind/src/resources/contour.yaml`
Build now passes again: `yarn build:extensions:kind`

Unit tests should pass as well:
`yarn test:extensions:kind`
